index.html

```diff
@@ -45,6 +45,15 @@ <h1 id="title">
         This is a work on progress.
       </p>
     </section>
+    <section id="introduction" class="informative">
+      <h2>Introduction
+      </h2>
+      <p>
+        The scope of this document is to specify the behavior of the 3D content as part of the
+        [=document=]. 3D content downloaded or obtained by some other mechanism
```

Comment:
Actually, I'm still a bit confused about this.... theoretically, all "fetching" needs to go through fetch regardless.